operator/bundle_helpers/rewrite.py

```diff
@@ -22,8 +24,10 @@ def rewrite(d, rewriter):
             if res is not None:
                 updates.append((k, res))
         for k, v in updates:
+            logging.info(f'Replaced: {d[k]} with {v}')
```

Comment:
We redirect the output and save it in the new CSV file. We shouldn't do that, rather we should have command-line option for input and output files.
Would this logging mess the `stdout` and appear in the middle of the output yaml?

Reply:
Seems like the output is driven to `stderr` by default as the output wasn't redirected here https://github.com/stackrox/stackrox/actions/runs/4025711606/jobs/6919288425#step:11:100 

BTW, I like how this logging looks. Maybe you can also do it for list replacement?

Reply:
Yes, `logging` doesn't use the same output as `print` by default. I've adde logging to list replacement and I think there was a bug there. But it did not have any impact because list was used for loop over the doc so no real replacement